### DISCUSSION TITLE
[@types/chrome]: chrome.webNavigation.WebNavigationFramedCallbackDetails lack certain properties

### DISCUSSION THREAD

**Discussion by chan-dev** [DefinitelyTyped/DefinitelyTyped#64385] [Issues with a @types package]
Here's the relevant documentation:

https://developer.chrome.com/docs/extensions/reference/webNavigation/#event-onCompleted

It lacks `documentLifecycle` and `frameType` just to name a few.

Haven't stumbled upon other missing types so far. Hopefully this will be added on the new version.

Thanks in advance.

[a comment by an automated account is not shown]

**Reply by moshikshechter** [reply]
I also bumped into this. is there a reason that `frameType` isn't defined?
